Documentation/security/policy/language.rst

```diff
@@ -503,6 +504,29 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../../examples/policies/l3/cidr/cidr.json
 
+.. _cidr_select_nodes:
+
+Selecting nodes with CIDR / ipBlock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../beta.rst
+
+By default, CIDR-based selectors do not match in-cluster entities (pods or nodes).
+Optionally, you can direct the policy engine to select nodes by CIDR / ipBlock.
+This requires you to configure Cilium with ``--policy-cidr-match-mode=nodes`` or
+the equivalent Helm value ``policyCIDRMatchMode: nodes``. It is safe to toggle this
+option on a running cluster, and toggling the option affects neither upgrades nor downgrades.
+
+When ``--policy-cidr-match-mode=nodes`` is specified, every agent allocates a
+distinct local :ref:`security identity <arch_id_security>` for all other nodes.
+This causes a minor increase in memory usage on very large clusters.
+
+This is particularly relevant to self-hosted clusters -- that is, clusters where
```

Comment:
@nathanjsweet / @zacharysarah want to review this paragraph?

Reply:
Thanks! LGTM.